sdks/java/io/elasticsearch-tests/elasticsearch-tests-common/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTestCommon.java

```diff
@@ -99,6 +103,9 @@ static String getEsIndex() {
   private static final long BATCH_SIZE = 200L;
   private static final long BATCH_SIZE_BYTES = 2048L;
 
+  public static final String UPDATE_INDEX = "partial_update";
+  public static final String UPDATE_TYPE = "test";
```

Comment:
these 2 statics are not needed if you rewrite the tests as suggested

Reply:
yes, it can be done like the other tests even with the index mapping overloading => I'll do it myself on another PR